lib/rules/declaration-block-no-shorthand-property-overrides/index.js

```diff
@@ -18,8 +18,13 @@ const rule = function (actual) {
       return
     }
 
-    root.walkRules(check)
-    root.walkAtRules(check)
+    root.walk((node) => {
+      if (node.type !== "rule" && node.type !== "atrule") {
+        return
+      }
+
+      check(node)
+    })
```

Comment:
It's kind of irrelevant. Benchmarks [here](https://github.com/stylelint/stylelint/pull/2354#issuecomment-280312263). Should I revert this change?